lightning/lightning_daemon.js

```diff
@@ -14,13 +18,21 @@ const {SSS_LND_TLS_CERT} = process.env;
   @returns
   <LND GRPC API Object>
 */
-module.exports = ({}) => {
+module.exports = ({network}) => {
   try {
-    return lightningDaemon({
-      cert: SSS_LND_TLS_CERT,
-      host: SSS_LND_GRPC_HOST,
-      macaroon: SSS_LND_MACAROON,
-    });
+    if (network === "testnet") {
```

Comment:
better to use a switch with an error for default indicating unknown network, also use single quotes